charts/ehrenamt-justiz/templates/secret.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ $secret.name }}
+  labels:
+    {{- include "getLabels" $data | trim  | nindent 4 }}
+  annotations:
+    helm.sh/resource-policy: keep
+data:
+  {{- range $key := $secret.keys }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)